starknet-foundry/crates/cast/tests/helpers/constants.rs

```diff
@@ -18,5 +18,7 @@ pub const MAP_CLASS_HASH_V2: &str =
     "0x06eb3fda9eaa7d99bb7c9e72c8d874988a5b9269f3f866773889bc5ab4ec9079";
 pub const DECLARE_TRANSACTION_HASH: &str =
     "0x03000a327c4651c8317a88000c11830e4988d94ed130e875987521d7b173763d";
+pub const ETH_CONTRACT_ADDRESS: &str =
```

Comment:
what is this cotnract exactly?

Reply:
This is contract which is used in payments on starknet. Fees are payed through it

Reply:
Cool, let's maybe add a more descriptive name to it then? something like `PAYMENTS_CONTRACT_ADDRESS` ?

Reply:
How about `FEE_CONTRACT_ADDRESS`?

Reply:
makes sense